app/src/main/java/com/rahobbs/todo/database/TodoCursorWrapper.java

```diff
@@ -0,0 +1,41 @@
+package com.rahobbs.todo.database;
+
+import android.database.Cursor;
+import android.database.CursorWrapper;
+
+import com.rahobbs.todo.helpers.TodoItem;
+import com.rahobbs.todo.database.TodoSchema.TodoTable;
+
+import java.util.Date;
+import java.util.UUID;
+
+/**
+ * Created by rachael on 2/9/16.
+ * Delegates calls to Cursor object
+ */
+public class TodoCursorWrapper extends CursorWrapper {
+    public TodoCursorWrapper(Cursor cursor){
+        super(cursor);
+    }
+
+    public TodoItem getTodoItem(){
```

Comment:
The whole point of this class is to allow you to instantiate a `TodoItem` from a `Cursor`. So rather than have a whole class around instantiating an object, I think you could move this as another constructor in the `TodoItem` POJO.